src/vscode/superbol-vscode-platform/superbol_cfg_explorer.ml

```diff
@@ -0,0 +1,284 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*                                                                        *)
+(*  Copyright (c) 2023 OCamlPro SAS                                       *)
+(*                                                                        *)
+(*  All rights reserved.                                                  *)
+(*  This source code is licensed under the MIT license found in the       *)
+(*  LICENSE.md file in the root directory of this source tree.            *)
+(*                                                                        *)
+(*                                                                        *)
+(**************************************************************************)
+
+open Vscode
+
+let read_whole_file filename =
+  (* open_in_bin works correctly on Unix and Windows *)
+  let ch = open_in_bin filename in
```

Comment:
I think there might be some exception to handle here (maybe re-raise, but `ch` might need to be closed anyways in this case).